src/patternfly/demos/PasswordStrength/examples/PasswordStrength.md

```diff
@@ -0,0 +1,175 @@
+---
+id: 'Password strength'
+beta: true
+section: demos
+---
+
+## Examples
+
+### Initial state
+```hbs
+  {{#> form}}
+    {{#> form-group}}
+      {{#> split split--modifier="pf-m-gutter"}}
```

Comment:
Looking at this split a little more, I don't know that we can do this in react. The `<FormGroup>` component takes a `label` for the label, and its children are passed into the form group control element. I'm wondering if we need to add a new form element for info to be displayed beside the label like this, and potentially support for what that would look like in a horizontal form layout. wdyt?